frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -8,7 +8,7 @@ import { isDefaultClass } from '../storage-class';
 
 /* Component StorageClassDropdown - creates a dropdown list of storage classes */
 
-class StorageClassDropdown_ extends React.Component<StorageClassDropdownProps, StorageClassDropdownState> {
+export class StorageClassDropdown_ extends React.Component<StorageClassDropdownProps, StorageClassDropdownState> {
```

Comment:
We should rename this if we are going to export. The trailing underscore implies it's internal.

Reply:
ack